### DISCUSSION TITLE
Next Stable Release

### DISCUSSION THREAD

**Discussion by RobEh** [sabnzbd/sabnzbd#2318] [General]
I am wondering if there will be a stable update being released anytime soon?  Current is 3.6.1.

Thanks,

**Comment by sanderjo** [comment]
> I am wondering if there will be a stable update being released anytime soon?

Can you define "anytime soon"?

**Comment by Safihre** [comment]
We just released the Release Candidate, so give or take 2 weeks.
Since no bugs were reported during alpha or beta phase and no changes were made to the code, the release candidate is already very stable.
